src/components/FAQItem.svelte

```diff
@@ -14,7 +14,7 @@
 
 <button
```

Comment:
The FAQ container is not centered on the screen.
<img width="1083" alt="Screenshot 2024-11-27 at 11 16 17 AM" src="https://github.com/user-attachments/assets/d6481866-1651-4efa-9959-643c1ef0f3dc">